content/posts/newsletter-020/index.md

```diff
@@ -64,6 +64,32 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### [Egregoria]
+
+![Egregoria city at dawn](egregoria.jpg)
+
+[Egregoria] ([GitHub][Egregoria], [Discord][egregoria-discord])
+by [@Uriopass]
+is a simulation oriented city builder that tries
+to replicate modern society as well as possible.
+
+The [8th devlog][egregoria-blog-post] was published.
+Updates include:
+
+ - Multiplayer based on deterministic lockstep
+ - Economy revamp inspired by Anno 1800
+ - Infinite world using procedural generation
+ - Many more QoL features
```

Comment:
```suggestion
- Multiplayer based on deterministic lockstep
- Economy revamp inspired by Anno 1800
- Infinite world using procedural generation
- Many more QoL features
```

Reply:
Ah, my bad! I wanted to install a markdown linter but Intellij has no such plugin.

Reply:
fixed. :-)